docs/docs/channels/chat/slack.md

```diff
@@ -6,42 +6,68 @@ This guide will walk you through the steps needed to obtain the `webhookUrl` tha
+   const novu = new Novu(process.env.NOVU_API_KEY);
 
-```typescript
-import { Novu, ChatProviderIdEnum } from '@novu/node';
+   const body = req.body; // From your HTTPS listener
+   await novu.subscribers.setCredentials('subscriberId', ChatProviderIdEnum.Slack, {
+     webhookUrl: body.incoming_webhook.url,
+   });
+   ```
 
-const novu = new Novu(process.env.NOVU_API_KEY);
+   - subscriberId is a custom identifier used when identifying your users within the Novu platform.
+   - providerId is a unique provider identifier. We recommend using our ChatProviderIdEnum to specify the provider.
+   - The third parameter is the credentials object. In this case we use the webhookUrl property to specify the webhook URL generated in the previous step.
 
-const body = req.body; // From your HTTPS listener
-await novu.subscribers.setCredentials('subscriberId', ChatProviderIdEnum.Slack, {
-  webhookUrl: body.incoming_webhook.url,
-});
-```
+   :::info
+   You need to set credentials for every subscriber because Slack generates a new Webhook URL on every new app install.
+   :::
 
-- `subscriberId` is a custom identifier used when identifying your users within the Novu platform.
-- `providerId` is a unique provider identifier. We recommend using our ChatProviderIdEnum to specify the provider.
-- The third parameter is the credentials object. In this case we use the `webhookUrl` property to specify the webhook URL generated in the previous step.
+4. Configure your Slack application as mentioned <span style={{textDecoration: "underline"}}>[below](/channels/chat/slack#slack-application-configuration)</span>.
+5. Add the `Add to Slack` button or the shareable URL to your application in order to request permission of access (scope: incoming-webhook).
+6. After the end-user finishes the authorization you will get the webhookUrl from the response of the OAuth under body.incoming_webhook.url, that you will use in step 3.
+7. You are all set up and ready to send your first chat message via our @novu/node package or directly using the REST API.
 
-:::info
-You need to set credentials for every subscriber because Slack generates a new Webhook URL on every new app install.
-:::
+</details>
 
-<!-- markdownlint-disable MD029 -->
+## Slack Application Configuration
 
-7. You are all set up and ready to send your first chat message via our `@novu/node` package or directly using the REST API.
-<!-- markdownlint-enable MD029 -->
+1. Go to OAuth & Permissions on Slack's Developer Dashboard and add your REDIRECT_URL in Redirect URLs.
+   - If you use self-hosted solution add the api endpoint you created on [step 1](/channels/chat/slack#self-hosted).
+   - If you use Novu hosted solution add https:<span/>//api.novu.co/v1/subscribers/.
+2. Go to Incoming Webhooks from the left menu and Activate Incoming Webhooks.
+3. Go to Manage Distribution and at the bottom of the page make sure to tick Remove Hard Coded Information and Activate Public Distribution.
```

Comment:
=> Go to Manage Distribution and at the bottom of the page, tick Remove Hard Coded Information and Activate Public Distribution.